types/zwjsbridge/tslint.json

```
@@ -0,0 +1,19 @@
+{ 
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "jsdoc-format": false,
+    "semicolon": false,
+    "interface-name": false,
+    "no-unnecessary-generics": false,
+    "interface-over-type-literal": false,
+    "max-line-length": false,
+    "void-return": false,
+    "npm-naming": false,
+    "no-empty-interface": false,
+    "no-consecutive-blank-lines": false,
+    "no-unnecessary-class": false,
+    "no-trailing-whitespace": false,
+    "no-const-enum": false,
+    "no-dead-reference": false
+  }
```

```suggestion
  "extends": "@definitelytyped/dtslint/dt.json"
```
- can you run:
```bash
npm run prettier -- --write ./types/zwjsbridge/*.ts
```
that will remove nearly all of the problems in your code:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes (formatting)

and, sorry to ask for this, I understand that character is perfectly OK under zh/ja/ko, replace
```ts
。*/
```
with:
```
 */
```
Looks like JSDoc linter fails when comment content ends with this specific character.